ui/src/components/content/catalog/services/ServiceDetail.tsx

```diff
@@ -0,0 +1,77 @@
+/*
+ * SPDX-License-Identifier: Apache-2.0
+ * SPDX-FileCopyrightText: Huawei Inc.
+ */
+
+import React from 'react';
+import { Descriptions, Divider, Space, Tag } from 'antd';
+import { CloudUploadOutlined, InfoCircleOutlined } from '@ant-design/icons';
+import { OclDetailVo } from '../../../../xpanse-api/generated';
+
+function ServiceDetail({
+    serviceDetails,
+    serviceRegions,
+}: {
+    serviceDetails: OclDetailVo;
+    serviceRegions: string[];
+}): JSX.Element {
+    return (
+        <>
+            <div className={'catalog-detail-class'}>
+                <h3>
+                    <CloudUploadOutlined />
+                    &nbsp;Available Regions
+                </h3>
+                <Space size={[0, 8]} wrap>
+                    {serviceRegions.map((serviceRegion, index) => (
+                        <Tag key={index} color='default'>
+                            {serviceRegion}
+                        </Tag>
+                    ))}
+                </Space>
+                <Divider />
+            </div>
+            <h3>
+                <InfoCircleOutlined />
+                &nbsp;Basic Information
+            </h3>
+            <Descriptions bordered column={1}>
+                <Descriptions.Item label='Property' labelStyle={{ width: '230px' }}>
```

Comment:
can the style be moved to css here also?

Reply:
leave as it is.